priv/repo/migrations/20161218125803_create_movie_details.exs

```diff
@@ -0,0 +1,21 @@
+defmodule MovieWagerApi.Repo.Migrations.CreateMovieRound do
+  use Ecto.Migration
+
+  def change do
+    create table(:movie_details) do
+      add :imdb_code, :string
+      add :poster_url, :string
+      add :release_date, :date
+      add :rating, :string
+      add :runtime, :string
+      add :genres, :string
+      add :summary, :string
+      add :director, :string
+      add :writers, :string
+      add :stars, :string
+      add :trailer_link, :string
+      add :movie_round_id, references(:movie_rounds)
```

Comment:
I'm thinking that maybe this relationship would work in the other direction.  I.e. `movie-round` should be aware of its corresponding `movie-details` but the details shouldn't care about the round information. Thoughts?